n2y/blocks.py

```diff
@@ -48,16 +49,17 @@ def __init__(self, client, notion_data, page=None, get_children=True):
         self.archived = notion_data['archived']
         self.notion_type = notion_data['type']
         self.notion_data = notion_data[notion_data['type']]
+        self.get_children = get_children
 
-        if get_children:
-            if self.has_children:
-                children = self.client.get_child_blocks(self.notion_id, page, get_children)
-            else:
-                children = []
+        if get_children and self.has_children:
+            children = self.client.get_child_blocks(self.notion_id, page, get_children)
         else:
-            children = None
+            children = Children(client=self.client)
         self.children = children
 
+    def replicate(self):
+        return self.client.wrap_notion_block(self.notion_block, self.page, self.get_children)
```

Comment:
When we replicate a block, perhaps we should remove the UUID from `self.notion_block` first, that way we don't have duplicate UUIDs anywhere?

Reply:
true, I deleted this method though as it's now dead code.